scss/_variables.scss

```diff
@@ -771,6 +771,7 @@ $modal-content-border-color:     rgba($black, .2) !default;
 $modal-content-border-width:     $border-width !default;
 $modal-content-box-shadow-xs:    0 .25rem .5rem rgba($black, .5) !default;
 $modal-content-box-shadow-sm-up: 0 .5rem 1rem rgba($black, .5) !default;
+$modal-content-border-radius:    $border-radius-lg !default;
```

Comment:
Thanks a lot for doing this change. 
Sorry, my bad, let's move this to line 721 so it's together with the rest of `modal-content-border` variables.

Reply:
Don't worry, I've just patch your PR with this change.